docs/docs/provide-devtools-config.md

```diff
@@ -0,0 +1,64 @@
+---
```

Comment:
this will be part of `with-redux.md`. You can add an additional chapter there, where you mention the possibility. Please use `bootstrapApplication` in the example instead of an `NgModule`.